wasmbinding/stargate_whitelist.go

```diff
@@ -73,7 +73,6 @@ func init() {
 	setWhitelistedQuery("/osmosis.epochs.v1beta1.Query/CurrentEpoch", &epochtypes.QueryCurrentEpochResponse{})
 
 	// gamm
-	setWhitelistedQuery("/osmosis.gamm.v1beta1.Query/NumPools", &gammtypes.QueryNumPoolsResponse{})
```

Comment:
Let's not remove these please to avoid breaking old contracts

Reply:
Brought them back for now, same question here, do we have plans to deprecate them in the future? I think V14 upgrade would be a good point to introduce this API break wdyt?

Reply:
We discussed this in another PR thread so I'm going to resolve this one